### PR TITLE
Markdown

### PR DESCRIPTION
**Pull request by IC-Socaciu**

### CODE REVIEW

File: pages/post/[interviewId].js [hunk 1]
```diff
@@ -111,16 +112,16 @@ export default function InterviewArticle() {
...
 }
 
 const TextContainer = styled.div`
   padding-left: 20px;
   margin-top: 0;
   word-break: normal;
-  margin-bottom: 120px;
-  margin-left: 20px;
+  margin-right: 10px;
+  margin-left: 10px;
```
Comment (CJAkkuec) [inline pages/post/[interviewId].js:124]:
```suggestion
  margin-inline: 10px;
```

File: pages/post/[interviewId].js [hunk 2]
```diff
@@ -144,19 +145,18 @@ const TitleContainer = styled.h2`
...
+  background-color: #fafafa;
 `;
 const ThumbUpIcon = styled(ThumbUps)`
   width: 20px;
   height: 20px;
   margin-right: 5px;
   fill: #95091b;
-  margin-left: 13px;
-  margin-bottom: 10px;
+  margin-left: 15px;
```
Comment (CJAkkuec) [inline pages/post/[interviewId].js:158]: You can use the margin-inline shorthand here as well.

**Review by CJAkkuec — CHANGES_REQUESTED** [review]
Just a couple of small changes 👍

File: pages/post/[interviewId].js (hunk 1 above)
Comment (IC-Socaciu) [reply pages/post/[interviewId].js:124]: 🙌 Thank you!

File: pages/post/[interviewId].js (hunk 2 above)
Comment (IC-Socaciu) [reply pages/post/[interviewId].js:158]: Done, thank you. :)

**Review by CJAkkuec — APPROVED** [review]
Ship it 🚀

**Pull request merged**